src/main/scala/tech/cryptonomic/conseil/util/Retry.scala

```diff
@@ -0,0 +1,133 @@
+package tech.cryptonomic.conseil.util
```

Comment:
This code is extremely readable, wow. We could learn something from this :(

Reply:
@vishakh

Reply:
@ba205 please elaborate more, I'm very interested to hear about possible improvements
What you feel here is more readable? What could we borrow as a general approach and use in other parts?